rfc6962-bis.xml

```diff
@@ -1412,21 +1424,39 @@ but it is expected there will be a variety.
             To reconstruct the TBSCertificate component of a precertificate from a certificate, TLS clients should:
             <list style="symbols">
               <t>
-                Remove the Transparency Information extension described in <xref target="x509v3_transinfo_extension"/>
+                Remove the Transparency Information extension described in <xref target="x509v3_transinfo_extension"/>.
               </t>
               <t>
-                Remove the Redacted Labels extension described in <xref target="redaction_extension"/>
+                If the Redacted SAN extension (<xref target="redacted_san_extension"/>) is present:
+                <list style="symbols">
+                  <t>TLS clients MUST verify it against the subjectAltName extension according to <xref target="verifying_redacted_san"/>.</t>
+                  <t>Once verified, remove the subjectAltName extension from the TBSCertificate.</t>
+                </list>
               </t>
-              <t>
-                Replace leftmost labels of each DNS-ID with <spanx style="verb">?</spanx>, based on the INTEGER value corresponding to that DNS-ID in the extension.
+            </list>
+          </t>
+          <t>
+            If the SCT checked is for a Precertificate (where the <spanx style="verb">type</spanx> of the <spanx style="verb">TransItem</spanx> is <spanx style="verb">precert_sct_v2</spanx>), then the client SHOULD also remove embedded v1 SCTs, identified by OID 1.3.6.1.4.1.11129.2.4.2 (See Section 3.3. of <xref target="RFC6962"/>), in the process of reconstructing the TBSCertificate. That is to allow embedded v1 and v2 SCTs to co-exist in a certificate (See <xref target="v1_coexistence"/>).
+          </t>
+        </section>
+        <section title="Verifying the Redacted SAN extension" anchor="verifying_redacted_san">
+          <t>
+            If the Redacted SAN extension is present, TLS clients MUST check that the SAN extension is present, that the SAN extension contains the same number of identifiers as the Redacted SAN extension, and that each identifier in the SAN extension has a matching identifier at the same position in the Redacted SAN extension. Two identifiers are matching if either:
```

Comment:
>  "same number of identifiers...matching identifier at the same position". Shouldn't this say "same number of dNSNames...matching label at the same position"?

No, that's the right wording.  dNSName matching is described a few lines down, and it's vital that non-dNSName SANs be checked so the CA can't omit them from the pre-certificate to evade monitoring.